src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -382,6 +385,20 @@ class device {
        return ret;
     }
 
+    uint32_t parseComputeUnitNum(const std::vector<std::string>& custat) const
+    {
+       uint32_t ret = 0;
```

Comment:
Please consider renaming the `ret` variable to something like `cu_count`.

Reply:
done